res/database/Default/user/therob.xml

```diff
@@ -2321,11 +2321,11 @@
 		<scripttemplate product="3" licence_type="1" guid="TheRob-Script-Show-GeburtdesSenders">
 			<title>
 				<de>Geburt des Senders</de>
-				<en>Birth of the channel</en>
+				<en>Birth of the station</en>
```

Comment:
you don't like the term "station"? Or you mean in general, some completely different title? Why?

Reply:
Alternative as in "in addition to" not "instead instead". As this script may appear several times, it means that "#2","#3" will be appended to the title.
It would be nice, if the script was offered using other titles.